htp/htp_response_generic.c

```diff
@@ -256,6 +256,17 @@ htp_status_t htp_process_response_header_generic(htp_connp_t *connp, unsigned ch
     htp_header_t *h_existing = htp_table_get(connp->out_tx->response_headers, h->name);
     if (h_existing != NULL) {
         // Keep track of repeated same-name headers.
+        if (h_existing->flags & HTP_FIELD_REPEATED) {
+            // This is the third occurence for this header.
+            if (!(h_existing->flags & HTP_FIELD_FOLDED)) {
```

Comment:
I don't think this folded flag should be used. Can we just rely on the HTP_FIELD_REPEATED flag? I think it's okay to allow just a single duplicate and that this should come with the warning already.

Reply:
Ok, the downside is that we will get 998 calls to `htp_log` if we have 1000 duplicate headers